### PR TITLE
Task/use store missing on put endpoint-parties

### PR DESCRIPTION
**Pull request by Jsyro**
First cut on store_missing put updates. If all the reqparser keys match the model attributes, then we can just loop through them. Quite a few formatting changes.

### CODE REVIEW

File: python-backend/app/api/parties/party/resources/party_resource.py [hunk 1]
```diff
@@ -32,26 +54,31 @@ class PartyResource(Resource, UserMixin, ErrorMixin):
...
     parser.add_argument(
         'address_line_2',
         type=str,
+        trim=True,
         store_missing=False,
         help='The second address line of the party address. Ex: 1234 Foo Road')
     parser.add_argument(
         'city',
         type=str,
+        trim=True,
```
Comment (nayr974) [inline python-backend/app/api/parties/party/resources/party_resource.py]: I didn't know you could set these to trim. Cool!

File: python-backend/app/api/parties/party/resources/party_resource.py [hunk 2]
```diff
@@ -146,28 +171,13 @@ def put(self, party_guid):
...
-            existing_party.address_line_2 = data.get(
-                'address_line_2') if 'address_line_2' in data else existing_party.address_line_2
-            existing_party.city = data.get('city') if 'city' in data else existing_party.city
-            existing_party.sub_division_code = data.get(
-                'sub_division_code'
-            ) if 'sub_division_code' in data else existing_party.sub_division_code
-            existing_party.post_code = data.get(
-                'post_code') if 'post_code' in data else existing_party.post_code
+            current_app.logger.info(f'Updating {existing_party} with {data}')
+            for key, value in data.items():
```
Comment (nayr974) [inline python-backend/app/api/parties/party/resources/party_resource.py]: 💯

File: python-backend/app/scheduled_jobs/ETL_jobs.py [hunk 3]
```diff
@@ -11,7 +11,7 @@ def _schedule_ETL_jobs(app):
 @register_apm
 def _run_ETL():
     with sched.app.app_context():
-        job_running = cache.get(ETL())
+        job_running = cache.get(ETL)
```
Comment (nayr974) [inline python-backend/app/scheduled_jobs/ETL_jobs.py]: Was this change intended? Does it have an effect?

Comment (Jsyro) [reply python-backend/app/scheduled_jobs/ETL_jobs.py]: @JustinMacaulay  requested this change, ETL isn't a function... so that would have been bad, it is unrelated to my changes.
